src/config.php

```diff
@@ -5,6 +5,8 @@
         'client' => env('SWISH_CLIENT_CERTIFICATE_PATH'),
         'password' => env('SWISH_CLIENT_CERTIFICATE_PASSWORD'),
         'root' => env('SWISH_ROOT_CERTIFICATE_PATH', true),
+        'signing' => env('SWISH_SIGNING_CERTIFICATE_PATH', null), // Optional, used for payouts
```

Comment:
What about wrapping these in a `storage_path` helper function or providing an example of how to add these in the `.env` file?

Reply:
@vinkla Yes, should be clarified that it's the full/absolute path of the file that is needed. Using `storage_path` is not optimal as the certificates might be stores outside of Laravel and/or the .env-file might be used by multiple apps in the same environment with different storage paths.